Update workfront-connector-install.md

somarc opened the pull request:
Update the public documentation to align with the outcome of ASSETS-25158 and SKYOPS-59018.

I also removed the section regarding Dispatcher since there is no Dispatcher in AEMaaCS in front of the AEM Author.  It's just the Fastly CDN. So no need to have a /filter rule or /clientheaders added.

anujkapo commented:
Tracked using CQDOC-17993.

aheim0 commented:
Thank you for highlighting this - the update has been applied.
Closing this issue.

aheim0 committed:
Merge branch 'main' into 2023.2.0-release

aheim0 committed:
Merge branch '2023.2.0-release' into Rel202302-ECFT-01

aheim0 committed:
PPUT-202303-01-01

aheim0 committed:
Merge pull request #3748 from AdobeDocs/PPUT-202303-01

PPUT-202303-01-01

Commit:
Fixed blank lines

There are spaces used instead of blank lines, which is causing bad output in LOC page.

bohnertchris committed:
Merge branch 'main' into blefebvr/SITES-12019-tense-mismatch

singhkh committed:
Summit book (#3749)

bohnertchris committed:
Merge branch 'main' into blefebvr/SITES-12019-tense-mismatch

bohnertchris committed:
Merge pull request #3747 from AdobeDocs/blefebvr/SITES-12019-tense-mismatch

SITES-12019 - [TBYB] Module 4 (React app) tense mismatch

singhkh committed:
Summit book (#3751)

bohnertchris committed:
Make public

bohnertchris committed:
Merge branch 'main' into CQDOC-19993

bohnertchris committed:
Change from Max

bohnertchris committed:
Merge pull request #3712 from AdobeDocs/CQDOC-19993

CQDOC-19993

gabrielwalt committed:
Update introduction.md

rickbrough committed:
Update image-profiles.md

rickbrough committed:
Merge pull request #3753 from AdobeDocs/rb021723

Update image-profiles.md

Ruchita454401 committed:
Update supporting-new-language-localization.md

Ruchita454401 committed:
Update supporting-new-language-localization.md

Ruchita454401 committed:
Merge pull request #3754 from AdobeDocs/updatelocalization

Update supporting-new-language-localization.md

chaik committed:
replaced March 1 with June 1

chaik committed:
added name of project purge OSGI config

raimanS committed:
Merge pull request #3757 from AdobeDocs/chaikels-patch-1

replaced March 1 with June 1

Commit:
CQDOC-20114 adding extra missing scope

aheim0 committed:
Merge branch 'main' into CFGQL-PQCHDM-02

aheim0 committed:
Merge branch 'main' into 2023.2.0-release

aheim0 committed:
Merge branch '2023.2.0-release' into Rel202302-ECFT-01

aheim0 committed:
CFGQL-PQCHDM-02-06

aheim0 committed:
CFGQL-PQCHDM-02-07

Commit:
CQDOC-20114 adding msft link

aheim0 committed:
Merge branch 'main' into patch-12

aheim0 committed:
Merge pull request #3750 from alp21312/patch-12

Fixed blank lines

sarchiz committed:
Merge branch 'main' into CQDOC-20114

sarchiz committed:
change link format

sarchiz committed:
Merge pull request #3758 from abhigarg/CQDOC-20114

CQDOC-20114 adding extra missing scope

bohnertchris committed:
CQDOC-20211

bohnertchris committed:
CQDOC-20212

bohnertchris committed:
CQDOC-20213

bohnertchris committed:
Merge pull request #3759 from AdobeDocs/bohnert

Various issues from GitHub

bohnertchris committed:
Merge branch 'main' into gwalt-patch-1

bohnertchris committed:
Small grammar corrections

aheim0 committed:
Merge branch 'main' into CFGQL-PQCHDM-02

aheim0 committed:
Merge branch 'main' into 2023.2.0-release

aheim0 committed:
Merge branch '2023.2.0-release' into Rel202302-ECFT-01

bohnertchris committed:
Merge pull request #3752 from AdobeDocs/gwalt-patch-1

Update introduction.md

bohnertchris committed:
Rename page, update links, add redirect

bohnertchris committed:
Updated steps

bohnertchris committed:
Updated adding steps

aheim0 committed:
Merge branch 'main' into CFGQL-PQCHDM-02

bohnertchris committed:
Typos

bohnertchris committed:
Merge pull request #3760 from AdobeDocs/CQDOC-20218

CQDOC-20218

aheim0 committed:
CFGQL-PQCHDM-02-HLC

aheim0 committed:
Merge branch 'main' into CFGQL-PQCHDM-02

aheim0 committed:
Merge branch 'main' into 2023.2.0-release

aheim0 committed:
Merge branch '2023.2.0-release' into Rel202302-ECFT-01

aheim0 committed:
CFGQL-PQCHDM-02-HLC01

bohnertchris committed:
Added Gabriel's publishing content with adjustments

bohnertchris committed:
Cross-linking and feedback from Max

bohnertchris committed:
Feedback from Max

bohnertchris committed:
Feedback from Max

bohnertchris committed:
Cross-linking

bohnertchris committed:
Typo

bohnertchris committed:
Merge pull request #3762 from AdobeDocs/bohnert

Adding details about publishing with Universal Editor

aheim0 committed:
GaTs-2023-01-01

aheim0 committed:
Merge pull request #3764 from AdobeDocs/GaTs-202303-01

GaTs-2023-01-01

raimanS committed:
update 1

raimanS committed:
update 2

guillaumecarlino committed:
Update release-notes-current.md

guillaumecarlino committed:
Update release-notes-2023-1-0.md

guillaumecarlino committed:
Update release-notes-current.md

guillaumecarlino committed:
Merge pull request #3766 from AdobeDocs/date

Adjusting release date

raimanS committed:
Merge branch 'main' into maintenance-update

guillaumecarlino committed:
Merge branch 'main' into 2023.2.0-release

raimanS committed:
Merge pull request #3765 from AdobeDocs/maintenance-update

update 1

raimanS committed:
update 1

raimanS committed:
Merge pull request #3767 from AdobeDocs/m-update

update 1

aheim0 committed:
Merge branch 'main' into CFGQL-PQCHDM-02

aheim0 committed:
Merge branch 'main' into 2023.2.0-release

aheim0 committed:
Merge branch '2023.2.0-release' into Rel202302-ECFT-01

aheim0 committed:
CFGQL-PQCHDM-02-HLC02

Commit:
Updated to 11382

Commit:
Typo correction

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Update 2023.1.0.md

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Merge branch 'main' into maintenance-release-11358

guillaumecarlino committed:
New video

aheim0 committed:
Merge branch '2023.2.0-release' into Rel202302-ECFT-01

aheim0 committed:
Merge pull request #3698 from AdobeDocs/Rel202302-ECFT-01

Rel202302 ecft 01

Commit:
Fixed dot placement

Commit:
Fixed dots placement

Commit:
Updated with informations from Release Digger

singhkh committed:
FormsCS: 65vsCS (#3722)

guillaumecarlino committed:
Update redirects.csv

guillaumecarlino committed:
Update TOC.md

guillaumecarlino committed:
Merge branch 'main' into maintenance-release-11358

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Update prerelease.md

guillaumecarlino committed:
Delete known-issues.md

guillaumecarlino committed:
Update home.md

guillaumecarlino committed:
Update prerelease.md

guillaumecarlino committed:
Update overview.md

guillaumecarlino committed:
Update prerelease.md

guillaumecarlino committed:
Update overview.md

singhkh committed:
Forms CS VS 65 (#3770)

guillaumecarlino committed:
Merge branch 'main' into maintenance-release-11358

guillaumecarlino committed:
Merge pull request #3713 from AdobeDocs/maintenance-release-11358

Maintenance release 11358

guillaumecarlino committed:
Merge branch 'main' into 2023.2.0-release

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Merge pull request #3771 from AdobeDocs/adding-KB

Adding kb article to maintenance release notes

guillaumecarlino committed:
Update TOC.md

guillaumecarlino committed:
Merge pull request #3772 from AdobeDocs/rename

Renaming TOC

lr345 committed:
Update release-notes-current.md

added prerelease channel feature - Export content fragments from AEM as a cloud service to Adobe target as JSON offers

aheim0 committed:
Merge branch 'main' into CFGQL-PQCHDM-02

aheim0 committed:
Merge pull request #3726 from AdobeDocs/CFGQL-PQCHDM-02

CFGQL-PQCHDM-02-01

guillaumecarlino committed:
Merge branch 'main' into 2023.2.0-release

guillaumecarlino committed:
Update release-notes-current.md

guillaumecarlino committed:
Update release-notes-current.md

aheim0 committed:
TCWI-202303-01-01

anujkapo committed:
Migrated the Publish videos to YouTube section from DM to Assets

aheim0 committed:
Merge pull request #3773 from AdobeDocs/TCWI-202303-01

TCWI-202303-01-01

anujkapo committed:
Merge branch 'main' into video-publishing-youtube

anujkapo committed:
Merge pull request #3774 from AdobeDocs/video-publishing-youtube

Migrated the Publish videos to YouTube section from DM to Assets

singhkh committed:
JDBC Support (#3778)

anujkapo committed:
Added a feature to the Assets section

anujkapo committed:
Removed the prerelease features section for Assets

anujkapo committed:
Minor updates to the Connected Assets article

anujkapo committed:
Merge pull request #3780 from AdobeDocs/connected-assets-april2023

Minor updates to the Connected Assets article

singhkh committed:
themes, templates, and form data models initial content (#3776)

singhkh committed:
Updated Reference assets for foundation components (#3781)

anujkapo committed:
Minor edits

bohnertchris committed:
Clarify requesting access process

bohnertchris committed:
Merge pull request #3782 from AdobeDocs/bohnert

Clarify requesting access process

bohnertchris committed:
Further detail

bohnertchris committed:
Merge branch 'main' into bohnert

bohnertchris committed:
Merge pull request #3783 from AdobeDocs/bohnert

UE access request detail

chaik committed:
one can deploy container packages

aheim0 committed:
Merge branch 'main' into 2023.2.0-release

sarchiz committed:
Merge pull request #3786 from AdobeDocs/chaikels-patch-2

one can deploy container packages

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Merge pull request #3787 from AdobeDocs/KI-11382

Known Issue 11382

blefebvre committed:
SITES-12520 - [TBYB] AEM Demo Assets GraphQL endpoint missing from trial environments (#3784)

Co-authored-by: David Gonzalez <dgonzale@adobe.com>

singhkh committed:
Forms - Release notes

singhkh committed:
Forms CS: Release Notes

singhkh committed:
minor changes

guillaumecarlino committed:
Update release-notes-2023-1-0.md

guillaumecarlino committed:
Update release-notes-current.md

guillaumecarlino committed:
Merge branch 'main' into 2023.2.0-release

kkour06 committed:
corrected typo

kkour06 committed:
Merge pull request #3790 from AdobeDocs/CQDOC-20030-k

corrected typo

guillaumecarlino committed:
Update release-notes-current.md

davidjgonzalez committed:
Update introduction-production-programs.md

guillaumecarlino committed:
Merge pull request #3791 from AdobeDocs/date

Adjusting Release Date

guillaumecarlino committed:
Merge branch 'main' into 2023.2.0-release

guillaumecarlino committed:
Update release-notes-2023-1-0.md

rickbrough committed:
Update dm-publish-settings.md

Added Important note about UGC vector assets limitation

rickbrough committed:
Merge branch 'main' into rb021723

rickbrough committed:
Update dm-publish-settings.md

rickbrough committed:
Merge branch 'rb021723' of https://git.corp.adobe.com/AdobeDocs/experience-manager-cloud-service.en into rb021723

rickbrough committed:
Merge pull request #3793 from AdobeDocs/rb021723

Update dm-publish-settings.md

atopper committed:
CQ-4352793 [CAM] Document that migrating to Rapid Development Environments (RDEs) is not supported
- remove lang=en, and some spelling mistakes and grammar suggestions

TylerRush committed:
Merge branch 'main' into dgonzale-patch-1

TylerRush committed:
Merge pull request #3792 from AdobeDocs/dgonzale-patch-1

Update introduction-production-programs.md

anujkapo committed:
Release notes updates

anujkapo committed:
fixed md errors

atopper committed:
Merge branch 'main' into top/CQ-4352793-rde

anujkapo committed:
PM feedback

anujkapo committed:
Merge pull request #3795 from AdobeDocs/workfront-enhanced-connector-1.9.9

Release notes updates

anujkapo committed:
Minor fix

anujkapo committed:
Merge branch 'main' into workfront-enhanced-connector-1.9.9

anujkapo committed:
Merge pull request #3796 from AdobeDocs/workfront-enhanced-connector-1.9.9

Minor fix

aheim0 committed:
GH79-202304-01-01

aheim0 committed:
Merge pull request #3797 from AdobeDocs/GH79-202304-01

GH79-202304-01-01

aheim0 committed:
Merge branch 'main' into patch-12

aheim0 committed:
Merge pull request #3769 from alp21312/patch-12

Fixed dots placement

bohnertchris committed:
CQDOC-20247

sarchiz committed:
Merge branch 'main' into top/CQ-4352793-rde

sarchiz committed:
Merge pull request #3794 from AdobeDocs/top/CQ-4352793-rde

CQ-4352793 [CAM] Document that migrating to Rapid Development Environ…

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Merge branch 'main' into known-issue

guillaumecarlino committed:
Merge pull request #3799 from AdobeDocs/known-issue

Adding known issues

aheim0 committed:
TTU-20230411-01-01

aheim0 committed:
Merge pull request #3800 from AdobeDocs/TTU-20230411-01

TTU-20230411-01-01

sarchiz committed:
Update go-live.md

sarchiz committed:
Merge pull request #3801 from AdobeDocs/sapatch-test

Update go-live.md

rickbrough committed:
Update video.md

Added APAC to DASH availability.

atopper committed:
CQ-4352819 [CAM] Remove duplicate AzCopy note

rickbrough committed:
Merge pull request #3802 from AdobeDocs/rb041123

Update video.md

chaik committed:
first row with sites and assets should have 2 RDEs

aheim0 committed:
Merge pull request #3805 from AdobeDocs/chaikels-patch-8

first row with sites and assets should have 2 RDEs

aheim0 committed:
CFC-20230412-01-01

aheim0 committed:
Merge pull request #3806 from AdobeDocs/CFC-20230412-01

CFC-20230412-01-01

guillaumecarlino committed:
Merge branch 'main' into 2023.2.0-release

sarchiz committed:
Merge branch 'main' into top/CQ-4352819-azcopy-notes

sarchiz committed:
Merge pull request #3803 from AdobeDocs/top/CQ-4352819-azcopy-notes

CQ-4352819 [CAM] Remove duplicate AzCopy note

sarchiz committed:
add security considerations docs

guillaumecarlino committed:
Merge branch 'main' into 2023.2.0-release

guillaumecarlino committed:
Update release-notes-current.md

guillaumecarlino committed:
Merge pull request #3702 from AdobeDocs/2023.2.0-release

2023.2.0 release

chaik committed:
RDE now available to all

aheim0 committed:
FF-20230413-01-01

aheim0 committed:
FF-20230413-01-02

aheim0 committed:
FF-20230413-01-03

aheim0 committed:
Merge pull request #3812 from AdobeDocs/FF-20230413-01

FF-20230413-01-01

aheim0 committed:
Merge branch 'main' into chaikels-patch-9

aheim0 committed:
Merge pull request #3810 from AdobeDocs/chaikels-patch-9

RDE now available to all

aheim0 committed:
CFET-202304130-01-01

aheim0 committed:
Merge pull request #3813 from AdobeDocs/CFET-20230413-01

CFET-202304130-01-01

guillaumecarlino committed:
Update release-notes-current.md

guillaumecarlino committed:
Merge pull request #3814 from AdobeDocs/date

Adjusting Release Date

sarchiz committed:
commit temp changes

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Merge pull request #3816 from AdobeDocs/important

Adding note about date discrepancy

blefebvre committed:
SITES-11845 - [TBYB] Build a learning guide with a simple CodePen app which demonstrates how to access content with the Headless SDK (#3811)

* SITES-11845 - [TBYB] Build a learning guide with a simple CodePen app which demonstrates how to access content with the Headless SDK

Stub in new code-playground trials page

* SITES-11845 - [TBYB] Build a learning guide with a simple CodePen app which demonstrates how to access content with the Headless SDK

* SITES-11845 - [TBYB] Build a learning guide with a simple CodePen app which demonstrates how to access content with the Headless SDK

* SITES-11845 - [TBYB] Build a learning guide with a simple CodePen app which demonstrates how to access content with the Headless SDK

Co-authored-by: David Gonzalez <dgonzale@adobe.com>

sarchiz committed:
Update security-considerations.md

sarchiz committed:
add note block

sarchiz committed:
Update security-considerations.md

sarchiz committed:
Merge branch 'main' into sapatch-12apr

TylerRush committed:
Merge branch 'main' into cm-release-2023.4.0

TylerRush committed:
Fixed syntax for link under Bug Fixes.

TylerRush committed:
Merge pull request #3798 from AdobeDocs/cm-release-2023.4.0

Cloud Manager release 2023.4.0

TylerRush committed:
Cleaning up Notes and Important notes.

blefebvre committed:
SITES-11845 - [TBYB] Build a learning guide with a simple CodePen app which demonstrates how to access content with the Headless SDK (#3817)

Include new code-playground.md page in TOC

TylerRush committed:
Update ingesting-content.md

TylerRush committed:
Merge branch 'main' into TRApril13

TylerRush committed:
Update ingesting-content.md

TylerRush committed:
Merge branch 'TRApril13' of https://git.corp.adobe.com/AdobeDocs/experience-manager-cloud-service.en into TRApril13

TylerRush committed:
Merge pull request #3818 from AdobeDocs/TRApril13

Cleaning up Notes and Important notes.

TylerRush committed:
Update ingesting-content.md

TylerRush committed:
Merge branch 'main' into TRApril13

TylerRush committed:
Update ingesting-content.md

TylerRush committed:
Merge branch 'TRApril13' of https://git.corp.adobe.com/AdobeDocs/experience-manager-cloud-service.en into TRApril13

TylerRush committed:
Merge pull request #3819 from AdobeDocs/TRApril13

Update ingesting-content.md

aheim0 committed:
CFT-20230414-01-01

aheim0 committed:
Merge pull request #3821 from AdobeDocs/CFT-20230414-01

CFT-20230414-01-01

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Update latest.md

guillaumecarlino committed:
Merge branch 'main' into critical-issue

guillaumecarlino committed:
Merge pull request #3822 from AdobeDocs/critical-issue

SKYOPS-56928  critical issue

guillaumecarlino committed:
Merge branch 'main' into sapatch-12apr

guillaumecarlino committed:
commenting section out

guillaumecarlino committed:
Merge pull request #3809 from AdobeDocs/sapatch-12apr

add security considerations docs

somarc committed:
Update workfront-connector-install.md

Update the public documentation to align with the outcome of ASSETS-25158 and SKYOPS-59018

guillaumecarlino commented:
Thanks @somarc, Anuj will review.
Tracked with CQDOC-20472.

guillaumecarlino commented:
Closing to align on CQDOC-20472 which was closed.
